Error compiling with 0.3

Seeing this error now when compiling complaining it can't find std and points at the FromSql portion. Worked fine with 0.2.3

```
error[E0433]: failed to resolve. Use of undeclared type or module `std`
 --> src/v1/models/pg_types.rs:1:30
  |
1 | #[derive(Debug, Copy, Clone, FromSql, ToSql)]
  |                              ^^^^^^^ Use of undeclared type or module `std`
```

Rust

``` rust
#[derive(Debug, Copy, Clone, FromSql, ToSql)]
#[postgres(name = "client_platform")]
pub enum ClientPlatform {
    #[postgres(name = "android")]
    Android,
    #[postgres(name = "ios")]
    Ios,
}
```

I have postgres 0.14, chrono 0.3, postgres-derive 0.3 all in my Cargo.toml, #[macro_use] extern crate for postgres and postgres-derive.

Very strange! I checked and the tests case I would've expected to fail for this works...

Any ideas on how to go about debugging this?

--- a/src/fromsql.rs
+++ b/src/fromsql.rs
@@ -59,7 +59,7 @@ fn enum_body(ident: &Ident, variants: &[Variant]) -> Tokens {
     let variant_idents = variants.iter().map(|v| &v.ident);
 
     quote! {
-        match std::str::from_utf8(buf)? {
+        match ::std::str::from_utf8(buf)? {
             #(
                 #variant_names => ::std::result::Result::Ok(#idents::#variant_idents),
             )*

